frontend/src/components/editProject/EditProjectRoot.tsx

```diff
@@ -226,13 +226,21 @@ export default function EditProjectRoot({
       locale: locale,
     })
       .then(function (response) {
+        const query = {
+          message: was_draft
+            ? texts.your_project_has_been_published_great_work
+            : texts.you_have_successfully_edited_your_project,
+        };
+        const matchedItem = response?.data?.hubUrl.find(
+          (item) => item.url_slug === project?.hubUrl
```

Comment:
Here we should use the hub url form the query string rather than from the project